src/view/conditions/hash.jsx

```diff
@@ -19,16 +19,16 @@ import MultipleItemEditor from './components/multipleItemEditor';
 import RegexToggle from '../components/regexToggle';
 
 const renderItem = field => (
-  <div data-row className="u-inlineBlock">
-    <label className="u-gapRight">
-      <span className="u-verticalAlignMiddle u-gapRight">Hash equals</span>
+  <div data-row className="u-inlineBlock u-flex">
+    <span className="u-flexCenter u-gapRight">Hash equals</span>
```

Comment:
We can't move the span outside the label without breaking accessibility (screen readers and such). Let me know if you need help figuring out an alternative.

Reply:
Done, to fix this we simply put a flex container inside a flex container.  in all my tests everything is rendering perfectly.